cime_config/acme/machines/config_batch.xml

```diff
@@ -51,6 +51,21 @@
      </submit_args>
    </batch_system>
 
+  <batch_system type="cobalt_theta" version="x.y">
```

Comment:
Why a new cobalt_theta section here instead of the cobalt definition already provided?